misc/python/materialize/feature_benchmark/scenario.py

```diff
@@ -58,16 +58,7 @@ def table_ten(self) -> TdAction:
         return TdAction(
             """
 > CREATE TABLE ten (f1 INTEGER);
-> INSERT INTO ten VALUES (0)
-> INSERT INTO ten VALUES (1)
-> INSERT INTO ten VALUES (2)
-> INSERT INTO ten VALUES (3)
-> INSERT INTO ten VALUES (4)
-> INSERT INTO ten VALUES (5)
-> INSERT INTO ten VALUES (6)
-> INSERT INTO ten VALUES (7)
-> INSERT INTO ten VALUES (8)
-> INSERT INTO ten VALUES (9)
+> INSERT INTO ten VALUES (0), (1), (2), (3), (4), (5), (6), (7), (8), (9)
```

Comment:
Our testing is not sophisticated enough,  but strictly speaking those are two different things -- one will produce 10 inputs at the same timestamp, the other 10 timestamps with 1 input each. Either way, all the benchmarking scenarios are already synthetic enough for this not to matter.